src/_cffi_src/openssl/callbacks.py

```diff
@@ -63,35 +106,34 @@
     }
 
     if (mode & CRYPTO_LOCK) {
-        PyThread_acquire_lock(_ssl_locks[n], 1);
+        mutex1_lock(_ssl_locks + n);
```

Comment:
It's not just a stylistic change.  `PyThread_type_lock` is a pointer type itself, and `PyThread_allocate_lock` was called in a loop, so `_ssl_locks` has changed here from being a `**` to being a `*`.  The memory is no longer being dereferenced; the locks are allocated contiguously.